bundles/org.eclipse.equinox.http.servlet/src/org/eclipse/equinox/http/servlet/internal/servlet/IncludeDispatchResponseWrapper.java

```diff
@@ -97,7 +97,6 @@ public void setLocale(Locale loc) {
 	}
 
 	@Override
-	@SuppressWarnings("deprecation")
```

Comment:
@jjohnstn the compiler here complains that this suppress warning is actually needed, can you take a look?